packages/test/test-end-to-end-tests/src/test/blobsisAttached.spec.ts

```diff
@@ -71,6 +87,39 @@ describeNoCompat("blob handle isAttached", (getTestObjectProvider) => {
 			directory.set(testKey, blob);
 			assert.strictEqual(blob.isAttached, true);
 		});
+
+		it("blob is acked after upload", async function () {
+			const testString = "this is a test string";
+			const testKey = "a blob";
+			const dataStore1 = await requestFluidObject<ITestFluidObject>(container, "default");
+
+			const map = await dataStore1.getSharedObject<SharedMap>(mapId);
+			// force write mode
+			await forceWriteMode(map, dataStore1);
+
+			const blob = await dataStore1.runtime.uploadBlob(stringToBuffer(testString, "utf-8"));
+			const pendingBlobs = (runtimeOf(dataStore1).getPendingLocalState() as any)
```

Comment:
can we avoid the any cast here, can't you just use the type? test can import types not exported by the packages

Reply:
we should fix it.